### DISCUSSION TITLE
Has anyone developped a plugin for the backend ?

### DISCUSSION THREAD

**Discussion by KawaKode** [Phalcode/gamevault-backend#295] [General]
I was wondering after stumbling upon the demo plugin repo if anyone has made or has started making a plugin for the backend.
I think about making a plugin allowing you to have a tiny web client to download games easilly but i do not know what we can do as there's not much documentation on plugins (or maybe I'm blind).

**Comment by Alfagun74** [comment]
Hi @KawaKode,

the plugin system is still very experimental and I explicit advised against developing a serious plugin for the time being, as it is a subject to change.

In fact I completely overhauled it just today in preparation for 13.0.0.

This version will include a fully worked out plugin system as well as more examples on how to develop them and some additional documentation.

Stay tuned. :)

**Comment by ykrasik** [comment]
Hi,

I stumbled upon this project by chance, but I implemented a similar app around... wow, I just realized it's 10 years ago. Its a desktop app in JavaFX (hey, 10 years ago).
I don't maintain it nowadays, but it has a few similar ideas and integrations with IGDB, GiantBomb and OpenCritic.
This is the link to the integration code, in case it saves anyone some time: https://github.com/ykrasik/gamedex/tree/master/provider

Use ChatGPT to adjust it to your repo :)

**Reply by Alfagun74** [reply]
Thanks!

**Comment by Alfagun74** [comment]
Hey @KawaKode, check out https://gamevau.lt/docs/server-docs/plugins/
